data/mods/My_Sweet_Cataclysm/sweet_mapgen/sweet_terrain.json

```diff
@@ -0,0 +1,38 @@
+[
+  {
+    "type": "terrain",
+    "id": "t_chocolate_sh",
+    "name": "shallow chocolate",
+    "description": "The chocolate isn't too deep here.  With a watertight container, you could gather fresh chocolate milk from here.  Not safe to drink as is.",
+    "symbol": "~",
+    "color": "light_blue",
+    "move_cost": 5,
+    "flags": [ "TRANSPARENT", "LIQUID", "NO_SCENT", "SWIMMABLE", "FISHABLE", "SHALLOW_WATER", "CHOCOLATE" ],
+    "connects_to": "WATER",
+    "examine_action": "water_source"
+  },
+  {
+    "type": "terrain",
+    "id": "t_chocolate_dp",
+    "name": "deep chocolate",
+    "description": "The chocolate isn't too deep here.  With a watertight container, you could gather fresh chocolate milk from here.  Not safe to drink as is.",
```

Comment:
```suggestion
    "description": "The chocolate isn't too deep here.  With a watertight container, you could gather what looks to be pure melted cacao.  This will not be tasty.",
```